vllm/v1/executor/ray_utils.py

```diff
@@ -117,6 +118,21 @@ def execute_model_ray(
                 # pickled.
                 if isinstance(output, AsyncModelRunnerOutput):
                     output = output.get_output()
```

Comment:
IIUC, this line will block the process, eliminating the gain of async scheduler, right?

Reply:
Yes, you are right!